docs/tutorials/jokes.md

```diff
@@ -2157,7 +2157,7 @@ We're going to handroll our own authentication from scratch. Don't worry, I prom
 
 Let's start by showing you our updated `prisma/schema.prisma` file. 💿 Go ahead and update your `prisma/schema.prisma` file to look like this:
 
-```tsx filename=prisma/schema.prisma lines=[13-20,24-25]
+```prisma filename=prisma/schema.prisma lines=[13-20,24-25]
```

Comment:
As described in https://github.com/remix-run/remix/commit/8b3fc2ecefd621e794038c4eeb00db1006063f64#commitcomment-66400057

https://github.com/remix-run/remix/runs/5136619644?check_suite_focus=true#step:6:1141

CC/ @mcansh